lcd.lua

```diff
@@ -186,7 +186,7 @@ local lcds = {
 }
 
 local reset_meta = function(pos)
-	minetest.get_meta(pos):set_string("formspec", "field[channel;Channel;${channel}]")
+	minetest.get_meta(pos):set_string("formspec", "field[channel;"..S("Channel")..";${channel}]")
```

Comment:
Please use the `core.formspec_escape` wrapper around `S` because translations might otherwise corrupt the formspec. For this string it's probably not an issue, but it's better to do it correct anyway.
Example wrapper function (if it does yet not exist):
```lua
-- formspec escape translation
local function FS(...)
	return core.formspec_escape(S(...))
end
```

Reply:
You're right. Thank you! Fixed. Check please.